### PR TITLE
Readme update 201912

### PR DESCRIPTION
**Pull request by Aylr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,100 +2,171 @@
...
-
-Quick Start
+Introduction
 --------------------------------------------------------------------------------
-[Getting Started](http://docs.greatexpectations.io/en/latest/getting_started.html) will teach you how to get up and running in minutes.
 
-For full documentation, visit [Great Expectations on readthedocs.io](http://great-expectations.readthedocs.io/en/latest/).
-
-[Down with Pipeline Debt!](https://medium.com/@expectgreatdata/down-with-pipeline-debt-introducing-great-expectations-862ddc46782a) explains the core philosophy behind Great Expectations. Please give it a read, and clap, follow, and share while you're at it.
+Great Expectations helps data teams eliminate technical debt, through data testing, documentation, and profiling.
```
Comment (Aylr) [inline README.md]:
```suggestion
Great Expectations helps data teams eliminate pipeline debt, through data testing, documentation, and profiling.
```

File: README.md [hunk 2]
```diff
@@ -2,100 +2,171 @@
...
 
-To make this project more accessible to all data practitioners, we are looking for user testers to help make the onboarding experience better! **If you are new to Great Expectations, in return for your time we’d love to help you get up and running!**
-Feel free to schedule here:  https://calendly.com/great-expectations-1/great-expectations-integration-one-on-one
+Software developers have long known that testing and documentation are essential for managing complex codebases. Great Expectations brings the same confidence, integrity, and acceleration to data science and engineering teams.
```
Comment (Aylr) [inline README.md]:
```suggestion
Software developers have long known that testing and documentation are essential for managing complex codebases. Great Expectations brings the same confidence, integrity, and acceleration to data science and data engineering teams.
```

File: README.md [hunk 3]
```diff
@@ -2,100 +2,171 @@
...
-Great Expectations helps teams save time and promote analytic integrity by offering a unique approach to automated testing: pipeline tests. Pipeline tests are applied to data (instead of code) and at batch time (instead of compile or deploy time). Pipeline tests are like unit tests for datasets: they help you guard against upstream data changes and monitor data quality.
 
-Software developers have long known that automated testing is essential for managing complex codebases. Great Expectations brings the same discipline, confidence, and acceleration to data science and engineering teams.
-
-
-Why would I use Great Expectations?
---------------------------------------------------------------------------------
+<!--
+--------------------------------------------------
+<<<A bunch of logos go here for social proof>>>
```
Comment (Aylr) [inline README.md:28]: ❤️

File: README.md [hunk 4]
```diff
@@ -2,100 +2,171 @@
...
+- `expect_column_values_to_not_be_null`
+- `expect_column_values_to_match_regex`
 - `expect_column_values_to_be_unique`
-- `expect_column_values_to_be_in_set`
-- `expect_column_mean_to_be_between`
-- ...and many more
+- `expect_column_values_to_match_strftime_format`
+- `expect_table_row_count_to_be_between`
+- `expect_column_median_to_be_between`
+- ...and [many more](https://docs.greatexpectations.io/en/latest/reference/expectation_glossary.html)
```
Comment (Aylr) [inline README.md:45]: maybe also bait the advanced users with one of the distribution expectations?

File: README.md [hunk 5]
```diff
@@ -2,100 +2,171 @@
...
+Expectations are <!--[declarative, flexible and extensible]()--> declarative, flexible and extensible.
+<!--To test out Expectations on your own data, check out the [<<step-1 tutorial>>]().-->
 
-**Automated data profiling**
+<!--
+<<animated gif showing typing an Expectation in a notebook cell, running it, and getting an informative result>>
+-->
 
-Writing pipeline tests from scratch can be tedious and counterintuitive. Great Expectations jump starts the process by providing powerful tools for automated data profiling. This provides the double benefit of helping you explore data faster, and capturing knowledge for future documentation and testing.
+**Full-stack data validation**
```
Comment (Aylr) [inline README.md]:
```suggestion
**Batteries-included data validation**
```

File: README.md [hunk 6]
```diff
@@ -2,100 +2,171 @@
...
+Check out [The Era of DIY Data Validation is Over]() for more details.
+-->
 
-Evaluating Expectations against data is just one step in a typical validation workflow. Great Expectations makes the followup steps simple, too: storing validation results to a shared bucket, summarizing results and posting notifications to slack, handling differences between warnings and errors, etc.
+<!--
+<<animated gif showing slack message, plus click through to validation results, a la: https://docs.google.com/presentation/d/1ZqFXsoOyW2KIkMBNij3c7KOM0RhajhAHKesdCL_BKHw/edit#slide=id.g6b0ff79464_0_183>>
+-->
 
-Great Expectations also provides robust concepts of Batches and Runs. Although we sometimes talk informally about validating "dataframes" or "tables," it’s much more common to validate batches of new data—subsets of tables, rather than whole tables. DataContexts provide simple, universal syntax to generate, fetch, and validate Batches of data from any of your DataSources.
+**Render tests to documentation**
```
Comment (Aylr) [inline README.md]:
```suggestion
**Tests are docs and docs are tests**
```

**Review by Aylr — COMMENTED** [review]
this is really really good

**Review by eugmandel — APPROVED** [review]

**Pull request merged**